set.mm

```diff
@@ -94027,8 +94048,8 @@ is that it denies the existence of a set containing itself ( ~ elirrv ).
   $( The Russell class is equal to the universe ` _V ` .  Exercise 5 of
      [TakeutiZaring] p. 22.  (Contributed by Alan Sare, 4-Oct-2008.) $)
   ruv $p |- { x | x e/ x } = _V $=
-    ( cvv weq cab cv wnel df-v equid elirrv nelir 2th abbii eqtr2i ) BAACZADAEZ
-    OFZADAGNPANPAHOOAIJKLM $.
+    ( cvv wtru cab cv wnel df-v tru elirr nelir 2th abbii eqtr2i ) BCADAEZNFZAD
+    AGCOACOHNNNIJKLM $.
```

Comment:
I checked whether elirrv was shorter, and it's only shorter in normal (expanded form), i.e. it doesn't qualify as shorter

In the process I found a shorter proof anyways which may be added optionally to this pr
```suggestion
    ( cvv cv wnel cab wcel vex elirr nelir 2th abbi2i eqcomi ) BACZMDZAENABMBFN
    AGMMMHIJKL $.
```
(TODO-SN: mathbox examples of non-qualifying shortenings)

Reply:
Please, add this shorter proof in a new MR, since we want that merged, while the fate of this MR is uncertain.

Then, you can resolve this conversation (like the other conversations you opened once you judge I have addressed the issue).